scripts/jobs/integrate/windows

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -x
+
+./pull-binary-libs.sh
+
+export ANT_OPTS="-Dfile.encoding=UTF-8 -server -XX:+AggressiveOpts -XX:+UseParNewGC -Xmx2G -Xss1M -XX:MaxPermSize=512M -XX:ReservedCodeCacheSize=128M"
+
+/home/tisue/apache-ant-1.9.6/bin/ant.bat \
```

Comment:
pushed an extra commit that makes this somewhat less gross, but only somewhat. it's commented TODO. let's discuss next week